### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by madey83** [DJATOM/x265-aMod#24] [Q&A]
Hello,
i have CPU family "Comet Lake" and i use StaxRip which file i should use:

x265-x64-v3.5+68-aMod-gcc12.2.1-opt-haswell.exe
x265-x64-v3.5+68-aMod-gcc12.2.1-opt-nehalem.exe
x265-x64-v3.5+68-aMod-gcc12.2.1-opt-sandybridge.exe
x265-x64-v3.5+68-aMod-gcc12.2.1-opt-skylake.exe
x265-x64-v3.5+68-aMod-gcc12.2.1-opt-znver1.exe
x265-x64-v3.5+68-aMod-gcc12.2.1-opt-znver2.exe
x265-x64-v3.5+68-aMod-gcc12.2.1-opt-znver3.exe
x265-x64-v3.5+68-aMod-gcc12.2.1.exe

**Comment by DJATOM** [comment]
Skylake should be fine. There are maybe new optimization sets availabe, but I don't have the electricity in near 17 hours/day, I can't even check it right now.

**Comment by madey83** [comment]
other quetion:

i use StaxRip with your latest version on x265 encoder and there is something on my 2017 LG OLED TV:

When i do encode existing DV profile 5/8 to reduce video size my TV can trigger and play new smaller file but i can't do FF or RW.
When i do FF or RW my TV start playing from begining.

this has been tested on other DV TV capable by other user.

When i do encode with NVENC there is no such behavior.
I have tested other x265 releases:

x265 binaries for Windows: 3.5+68-40e37bc
x265-3.5+37+12-4e46995bc-[Mod-by-Patman]

**Comment by DJATOM** [comment]
You should check for that against official x265 builds and if behavior is the same, [open issue on their bitbucket repo](https://bitbucket.org/multicoreware/x265_git/issues?status=new&status=open).

**Comment by madey83** [comment]
where i can find official binaries version of the source code?  i have no idea how to compile the code.
i tested this issue againt this: x265 binaries for Windows: 3.5+68-40e37bc [GCC 12.2 ] donloaded from http://msystem.waw.pl/x265/

**Reply by DJATOM** [reply]
Your version is fine I guess. There's only one new commit added in official repo, and unofficial build here: https://forum.doom9.org/showthread.php?p=1978218#post1978218

**Comment by madey83** [comment]
Hi,

I have tested this build with option --no-open-gop in x265 and this solve my problem.
